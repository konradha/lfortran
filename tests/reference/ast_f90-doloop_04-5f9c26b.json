{
    "basename": "ast_f90-doloop_04-5f9c26b",
    "cmd": "lfortran --show-ast-f90 --no-color {infile}",
    "infile": "tests/../integration_tests/doloop_04.f90",
    "infile_hash": "a8d3b1711d8ac0c6351904c8e1e1d34527d879987044335492d5ad61",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast_f90-doloop_04-5f9c26b.stdout",
    "stdout_hash": "e19f78af43d21ad32cd72d37dd7854d6f5b4271ba8ab18458ee1f509",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}