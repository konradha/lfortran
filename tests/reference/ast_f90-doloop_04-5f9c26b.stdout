program doloop_04
implicit none
integer :: i, j
j = 0
a: do i = 1, 10
    j = j + i
    if (i == 2) then
        exit a
    end if
end do a
if (j /= 3) then
    error stop
end if

j = 0
b: do i = 1, 10, 2
    j = j + i
    if (i == 3) then
        exit b
    end if
end do b
if (j /= 4) then
    error stop
end if

j = 0
i = 1
c: do
    j = j + i
    if (i == 2) then
        exit c
    end if
    i = i + 1
end do c
if (j /= 3) then
    error stop
end if

end program doloop_04
